BUILD_DETAIL.md

```diff
@@ -97,23 +97,22 @@ $ bundle exec yard doc --no-cache
 $ bin/yard doc --no-cache
 ```
 
-## RuboCop
+## Rubocop
 
-We use [RuboCop](https://github.com/rubocop-hq/rubocop) to enforce style
-conventions on the project so that the code has stylistic consistency
-throughout. Run with:
+We use [Rubocop](https://github.com/bbatsov/rubocop) to enforce style conventions on the project so
+that the code has stylistic consistency throughout. Run with:
 
 ```
-$ bundle exec rubocop
+$ bundle exec rubocop lib
```

Comment:
This is the only regression worth fixing, along with a similar change in `DEVELOPMENT.md`.
Other changes being overwritten do not matter much.